src/oom/oomd-conf.c

```diff
@@ -0,0 +1,119 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "conf-parser.h"
+#include "log.h"
+#include "oomd-conf.h"
+#include "oomd-manager.h"
+
+static int arg_swap_used_limit_permyriad = -1;
+static int arg_mem_pressure_limit_permyriad = -1;
+static usec_t arg_mem_pressure_usec = DEFAULT_MEM_PRESSURE_DURATION_USEC;
```

Comment:
these no longer need to be global variables, as they are used from the manager object, so they can be defined in the manager_parse_config_file() function below instead

Reply:
Done, but I had to get rid of the `static` for the ConfigTableItem array in `manager_parse_config_file`